lib/src/widgets/common/generic_dialog.dart

```diff
@@ -16,30 +14,35 @@ import 'package:fa_flutter_ui_kit/src/utils/log_utils.dart';
 ///           },
 ///           leftActionText: 'No',
 ///         ));
-
 import 'package:flutter/material.dart';
 
 class GenericDialog extends StatelessWidget {
-  const GenericDialog(
-      {Key? key,
-      this.title,
-      this.subtitle,
-      this.leftActionText,
-      this.rightActionText,
-      this.leftButtonFunction,
-      this.rightButtonFunction,
-      this.buttonBgColor = Colors.transparent,
-      this.buttonTextBgColor})
-      : super(key: key);
+  const GenericDialog({
+    Key? key,
+    this.title,
+    this.subtitle,
+    this.leftActionText,
+    this.rightActionText,
+    this.leftButtonFunction,
+    this.rightButtonFunction,
+    this.subdescription,
+    this.buttonBgColor = Colors.transparent,
+    this.buttonTextBgColor,
+    this.rightTextColor,
```

Comment:
Add for left button also

Reply:
buttonBgColor is for left button, If name is changed then other project will get affected.